### PR TITLE
Change the Navbar design and links

### PR DESCRIPTION
**Pull request by Piumal1999**
## Purpose

Resolving issue #290

## Goals
This will make the navbar visible on white backgrounds
And correct the links

## Approach
Changed the style of the navbar and added the links

### Screenshots
Join_Us_-_2019-12-22_23 22 46

  
### Preview Link
https://piumal1999.github.io/sef-site/src/main/webapp/join_us.html
(This can be seen in any page)

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
N/A

## Test environment
Ubuntu 19.04
Google Chrome

## Learning
N/A

### CODE REVIEW

File: src/main/webapp/assets/content/static/navbar.html [hunk 1]
```diff
@@ -1,165 +1,145 @@
...
-                            </a>
-                            <div class="dropdown-menu">
-                                <a href="https://sefglobal.org/about/about.html" class="dropdown-item">Who we are</a>
-                                <a href="https://sefglobal.org/about/contribute.html" class="dropdown-item">Contribute</a>
-                                <a href="https://sefglobal.org/about/contact.html" class="dropdown-item">Contact</a>
-                                <a href="https://sefglobal.org/about/team.html" class="dropdown-item">Team</a>
-                                <a href="https://sefglobal.org/about/alumni.html" class="dropdown-item">Alumni</a>
+                                </a>
+                                <a href="irc.html" class="media d-flex align-items-center">
+                                    <img src="https://sefglobal.org/assets/img/brand/irc.png"
```
Comment (jayasanka-sack) [inline src/main/webapp/assets/content/static/navbar.html]: Change image sources to local

Comment (Piumal1999) [reply src/main/webapp/assets/content/static/navbar.html]: Done

**Comment by jayasanka-sack** [comment]
Preview link is wrong

**Comment by Piumal1999** [comment]
Added the correct preview link

**Comment by jayasanka-sack** [comment]
merged

**Pull request merged**